### ISSUE TITLE
real_value = True

### ISSUE THREAD

**Issue by huiguhean** [LeiBAI/AGCRN#16]
if self.args.real_value:
                label = self.scaler.inverse_transform(label)#这怎么只反转目标值，不反转预测值，用标准化后的值预测真实值？
            loss = self.loss(output.cuda(), label)
